### PR TITLE
changed gtesest and gmock to be statically linked

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

**Comment by ExtraGravity** [comment]
I'll update the travis to install google test once you've looked at this.

File: Tests/Tests.pro [hunk 1]
```diff
@@ -2,7 +2,7 @@ TEMPLATE = app
 QT += testlib
 CONFIG += testcase
 
-LIBS += -L../../build/.lib -LC:\gtest\lib -lViewLayer -lBusinessLayer -lCommunicationLayer -lPresenterLayer -lDataLayer -lgmock -lgtest
+LIBS += -L../../build/.lib -LC:\gtest\lib -lViewLayer -lBusinessLayer -lCommunicationLayer -lPresenterLayer -lDataLayer libgmock.a
```
Comment (JordanHeinrichs) [inline Tests/Tests.pro:5]: Change this line to UNIX only. Remove -LC:\gtest\lib.

For now on windows you could see if you can just disable the test building unless you can get it working.

File: README.adoc [hunk 2]
```diff
@@ -7,3 +7,28 @@ The program also rebroadcasts the data feed via a multicast group on the local n
...
+3. Compile googletest:
+    `g++ -isystem googletest/include/ -Igoogletest -isystem googlemock/include/ -Igooglemock -pthread -c googletest/src/gtest-all.cc`
+
+4. Compile googlemock:
+    `g++ -isystem googletest/include/ -Igoogletest -isystem googlemock/include/ -Igooglemock -pthread -c googlemock/src/gmock-all.cc`
+
+5. Collect both the resulting .o files into an archive file:
+    `ar -rv libgmock.a gtest-all.o gmock-all.o`
+
+Congratulations! You now have a `libgmock.a` archive file, now include it in the `Schulich-Delta-Desktop-Client-Telemetry/Tests/` directory and you should be good to go, run `make` in the test directory to check if you can compile the tests.
```
Comment (JordanHeinrichs) [inline README.adoc:34]: Hmmm, I am not a huge fan of having built libraries next to the source.  What about putting it in build/.lib? The qmake file should not need to change.

File: Tests/Tests.pro (hunk 1 above)
Comment (ExtraGravity) [inline Tests/Tests.pro:5]: I'll get it to work on Windows as well

File: README.adoc (hunk 2 above)
Comment (ExtraGravity) [inline README.adoc:34]: Can we put the build and release directory in the same place as the project but ignored in git? I have always found it a little awkward for the release and build folders to be in a higher directory than the git repo.

The qmake file doesn't change (I'm assuming you're talking about the generated makefile), only the .pro file needs to change.

Comment (JordanHeinrichs) [inline README.adoc:34]: I agree. In all my new projects that is the current setup. Basically I have project/src and project/build. The .pro file doesn't even need to change, just move everything from the current repo one folder down (That is if you are fine with building only from the src repo).

**Pull request closed without merging**